components/textbook-beta/TextbookBetaHeader.vue

```diff
@@ -131,6 +136,36 @@ export default class TextbookBetaHeader extends (Vue as VueConstructor<VueCompon
     }
   }
 
+  &__logo {
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    width: 100%;
+    padding-top: 100%;
+    transform: translate(-50%, -50%);
+
+    @include mq($until: large) {
+      width: calc(100% + 5rem);
```

Comment:
can we use `$spacing-11`? [reference Carbon](https://www.carbondesignsystem.com/guidelines/spacing/overview/#spacing-scale)

Reply:
Done!